apps/admin/backend/src/store.ts

```diff
@@ -1705,6 +1705,13 @@ export class Store {
     const officialCandidateNameLookup =
       getOfficialCandidateNameLookup(election);
 
+    // we ignore unmarked write-ins that are not adjudicated for a candidate
+    whereParts.push(`not (
+      write_ins.is_unmarked = 1 and
```

Comment:
Thanks for adding the comment, it's more clear now.

I'm a bit concerned that the 0 changed to a 1 but I don't see a corresponding test update. Is this case covered?